src/libraries/Microsoft.Extensions.Caching.Memory/src/CacheEntry.cs

```diff
@@ -159,9 +217,9 @@ public void Dispose()
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)] // added based on profiling
-        internal bool CheckExpired(in DateTimeOffset now)
-            => _state.IsExpired
-                || CheckForExpiredTime(now)
+        internal bool CheckExpired(DateTime utcNow)
```

Comment:
Why did we remove `in` here and below?

Reply:
DateTime fits in a single register (unlike DateTimeOffset).